tests/setup/mod.rs

```diff
@@ -163,16 +186,21 @@ pub mod nigiri {
 
         start_nigiri();
         start_lspd();
+        start_rgs();
     }
 
     pub fn stop() {
+        debug!("RGS server stopping ...");
```

Comment:
would it make sense to move the debug log to the `stop_rgs` function, to keep it consistent with the `start_rgs` function?